docs/latest/advanced/extensions.md

```diff
@@ -140,8 +140,8 @@ children: PropTypes.node,
 customProps: PropTypes.object
 ```
 
-Viewport components are managed by the `LayoutManager`. Which Viewport component
```

Comment:
Why rename something that's existed for such a long time?

You may have refactored how it operates, but the `ViewportGrid` is essentially the replacement for the layout manager. It manages layout, so why not keep the old name?